classes/classes/Saves.as

```diff
@@ -1792,18 +1792,28 @@ public function loadGameObject(saveData:Object, slot:String = "VOID"):void
 		player.hipRating = saveFile.data.hipRating;
 		player.buttRating = saveFile.data.buttRating;
 		
+		if (player.lowerBody > 3 && player.lowerBody < 5) { //Bitch's way of saying 4. So we can delete LOWER_BODY_TYPE_CENTAUR
```

Comment:
If you notice, 4 isn't an entry in appearanceDefs anymore. I was skirting around directly saying 4 unless you want to throw errors.